src/main/java/com/bieganski/ox/App.java

```diff
@@ -1,22 +1,28 @@
 package com.bieganski.ox;
 
+import com.bieganski.ox.user_interface.ConsoleInterface;
+import com.bieganski.ox.user_interface.UserInterface;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.util.Scanner;
 
-class App {
+/**
+ * Main class for OX application
+ * @author Kamil Biegański
+ */
+public class App {
+    public static final Logger LOG = LoggerFactory.getLogger(App.class);
+
     public static void main(String[] args) {
+
+        LOG.info("Application started");
+
         Scanner scanner = new Scanner(System.in);
         UserInterface userInterface = new ConsoleInterface(scanner, System.out);
         userInterface.println("Tic tac toe!");
-        //Const size, just for checking if works
-        Board board = new Board(10);
-        board.addListener(userInterface);
-        board.updateListeners();
-      
-        PositionCalculator positionCalculator = new PositionCalculator(new PositionValidator(),10);
-      
-        String input = userInterface.askForInput();
-        if (new PositionValidator().checkString(input)) {
-            board.addField(new Field(positionCalculator.calculatePosition(input), Symbol.X));
-        }
+        //TODO: change const values ,create game setup
+        GameFlow gameFlow = new GameFlow(userInterface, Integer.parseInt(args[0]));
+        new Thread(gameFlow).start();
```

Comment:
Integer.parseInt(args[0]) can return exception. Maybe you should handle it before put to constructor ?

Reply:
It's temporary